ci-operator/step-registry/strimzi/OWNERS

```diff
@@ -0,0 +1,11 @@
+approvers:
```

Comment:
This isn't a must-have, but it may be nice for you and your team to utilize the alias functionality in OWNERS files. You can create on in the [`OWNERS_ALIASES` file](https://github.com/openshift/release/blob/master/OWNERS_ALIASES) if you'd like.

Reply:
will check it, thanks